rllib/BUILD

```diff
@@ -1073,7 +1073,7 @@ py_test(
     name = "test_rollout",
     main = "tests/test_rollout.py",
     tags = ["tests_dir", "tests_dir_R"],
-    size = "large",
+    size = "enormous",
```

Comment:
@sven1977 not sure why this is taking so long, but setting it to "enormous" size for now as a workaround :)